### PR TITLE
feat : customer360 unhappy Path handles (CXSPA-3297)

### PR DESCRIPTION
**Pull request by kimhw0630**
Closes: https://jira.tools.sap/browse/CXSPA-3297

### CODE REVIEW

File: feature-libs/asm/customer-360/assets/translations/en/customer-360.ts [hunk 1]
```diff
@@ -6,6 +6,9 @@
 
 export const customer360 = {
   customer360: {
+    alertErrorMessage:
+      'The information cannot be be loaded. Please try again later or contact your  system administrator.',
+    errorMessageHeader: 'Oops! Something didn\x27t',
```
Comment (RadhepS) [inline feature-libs/asm/customer-360/assets/translations/en/customer-360.ts:11]: Is this error message complete?

File: feature-libs/asm/customer-360/assets/translations/en/customer-360.ts [hunk 2]
```diff
@@ -6,6 +6,9 @@
 
 export const customer360 = {
   customer360: {
+    alertErrorMessage:
+      'The information cannot be be loaded. Please try again later or contact your  system administrator.',
```
Comment (RadhepS) [inline feature-libs/asm/customer-360/assets/translations/en/customer-360.ts]: Small comment but there is an extra space between "your" and "system administrator"

File: feature-libs/asm/customer-360/assets/translations/en/customer-360.ts (hunk 1 above)
Comment (kimhw0630) [reply feature-libs/asm/customer-360/assets/translations/en/customer-360.ts:11]: yes according to UX..

File: feature-libs/asm/customer-360/assets/translations/en/customer-360.ts (hunk 2 above)
Comment (kimhw0630) [reply feature-libs/asm/customer-360/assets/translations/en/customer-360.ts]: oh ... thanks..

**Review by RadhepS — APPROVED** [review]

**Pull request merged**